### ISSUE TITLE
Error setting up entry Senertec for senertec - after HA Update to Version 2023.5.3

### ISSUE THREAD

**Issue by MarAlze** [Kleinrotti/hass-senertec#5]
Good evening,

I have updated HA today as written above and after restarting unfortunately this error occurred.

If you need more information contact me.

```
ValueError: Sensor sensor.windrichtung has device class None, state class None unit  and suggested precision None thus indicating it has a numeric value; however, it has the non-numeric value: N (<class 'str'>)
2023-05-25 15:24:37.395 INFO (SyncWorker_5) [senertec.client] Logging in..
2023-05-25 15:24:38.539 INFO (SyncWorker_5) [senertec.client] Login was successful.
2023-05-25 15:24:38.540 INFO (SyncWorker_5) [senertec.client] Initializing senertec platform...
2023-05-25 15:24:39.910 INFO (senertec-websocket) [senertec.client] Connected to Senertec websocket
2023-05-25 15:24:44.070 WARNING (senertec-websocket) [senertec.client] No enum translation found for datapoint 'Gerätstatus'.
2023-05-25 15:24:47.518 INFO (SyncWorker_0) [senertec.client] Logging out..
2023-05-25 15:24:48.099 INFO (senertec-websocket) [senertec.client] Senertec Websocket closed with code None
2023-05-25 15:24:48.138 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry Senertec for senertec
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 387, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/senertec/__init__.py", line 51, in async_setup_entry
    hass.config_entries.async_setup_platforms(entry, PLATFORMS)
AttributeError: 'ConfigEntries' object has no attribute 'async_setup_platforms'
2023-05-25 15:24:54.300 INFO (SyncWorker_5) [senertec.client] Logging in..
2023-05-25 15:24:55.281 INFO (SyncWorker_5) [senertec.client] Login was successful.
2023-05-25 15:24:55.282 INFO (SyncWorker_5) [senertec.client] Initializing senertec platform...
2023-05-25 15:24:56.313 INFO (senertec-websocket) [senertec.client] Connected to Senertec websocket
2023-05-25 15:24:59.121 WARNING (senertec-websocket) [senertec.client] No enum translation found for datapoint 'Gerätstatus'.
2023-05-25 15:25:03.489 INFO (SyncWorker_10) [senertec.client] Logging out..
2023-05-25 15:25:03.944 INFO (senertec-websocket) [senertec.client] Senertec Websocket closed with code None
2023-05-25 15:25:03.958 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry Senertec for senertec
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 387, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/senertec/__init__.py", line 51, in async_setup_entry
    hass.config_entries.async_setup_platforms(entry, PLATFORMS)
AttributeError: 'ConfigEntries' object has no attribute 'async_setup_platforms'

```

**Comment by Kleinrotti** [comment]
Seems like something got changed since HA version 2023.5.
I need to look into it. Until then, a rollback to a version prior 2023.5. is the only solution to get the integration working again.

**Comment by Kleinrotti** [comment]
Fixed in version 0.1.5.
